tsconfig.json

```diff
@@ -12,7 +12,7 @@
         "experimentalDecorators": true,
         "moduleResolution": "node",
         "types": ["node"],
-        "lib": ["es2017", "dom"],
+        "lib": ["es2018", "dom"],
```

Comment:
Is it on purpose 🧐?

Reply:
Yes sir. Rocket.Chat's version of NodeJS got updated to v12 :)